dd-sdk-android/src/test/kotlin/com/datadog/android/utils/DatadogExt.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2019 Datadog, Inc.
+ */
+
+package com.datadog.android.utils
+
+import android.content.Context
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import com.datadog.android.BuildConfig
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+
+/**
+ * Mocks a context with the minimal behavior to initialize the Datadog library.
+ */
+fun mockContext(
+    fakePackageName: String = BuildConfig.LIBRARY_PACKAGE_NAME,
+    fakePackageVersion: String = BuildConfig.VERSION_NAME
+): Context {
+    val mockPackageInfo = PackageInfo()
+    val mockPackageMgr = mock<PackageManager>()
```

Comment:
Do we have a convention about using abbreviations? I don't find `mockPackageManager` that wrong, whereas `*Mgr` seems inconsistent in this file.